fi.csc.Elmer.yaml

```
@@ -0,0 +1,453 @@
+        sha256: e5b57537c741400cf6134b428fc1689a649d7d38d9bb9c1b6d64f092ea28178a
+        x-checker-data:
+          type: anitya
+          project-id: 227581
+          stable-only: true
+          url-template: https://github.com/oneapi-src/oneTBB/archive/refs/tags/v$version.tar.gz
+
+  - name: Trilinos
+    buildsystem: cmake
+    builddir: true
+    build-options:
+      cflags: -fPIC
+      cxxflags: -fPIC
+    config-opts:
+      - -DTrilinos_ENABLE_TESTS=OFF
+      - -DTrilinos_ENABLE_ML=ON
+      - -DTrilinos_ENABLE_Belos=ON
+      - -DTrilinos_ENABLE_Epetra=ON
+      - -DTPL_ENABLE_MPI:BOOL=ON
+    sources:
+      - type: git
+        url: https://github.com/trilinos/Trilinos.git
+        tag: trilinos-release-13-2-0
+        commit: 4a5f7906a6420ee2f9450367e9cc95b28c00d744
+        x-checker-data:
+          type: git
+          tag-pattern: ^(v[\d.]+)$
+    cleanup:
+      - /lib/cmake/tribits/examples
+      - /lib/cmake/tribits/doc
+
+  #- name: espreso
+    #buildsystem: simple
+    #build-commands:
+      #- ls -l
+      #- cp -r include/* /app/include
+    #sources:
+      #- type: git
+        #url: https://github.com/It4innovations/espreso
+        #commit: 5dbd1d4377955308dc2fed94a898098bbf13878c
+
+  #- name: MUMPS
+    #buildsystem: simple
+    #build-options:
+      #env:
+        #CXXFLAGS: -L/app/lib
+    #build-commands:
+      #- rm -r examples # No need examples
+      #- sed -i 's:(cd\ examples;\ $(MAKE)\ clean)::g' Makefile # Don't build example
+      #- sed -i 's:cd\ examples;\ $(MAKE)\ all::g' Makefile # Don't build example
+      #- sed -i 's:/usr:/app:g' Makefile.inc # Adjust prefix to Flatpak
+      #- sed -i 's:/include/scotch:/include:g' Makefile.inc # Adjust SCOTCH include dir
+      #- sed -i 's:/include/metis:/include:g' Makefile.inc # Adjust metis include dir
+      #- sed -i 's:/-lmpi:/-lMPI:g' Makefile.inc # Adjust MPI library case
+      ##- ld -L/app/lib -lmpi --verbose
+      ##- ld -lmpi --verbose
+      #- make all
+      #- cd include && install -D -m644 *.h "/app/include"
+      #- cd lib && install -D -m644 *.so "/app/lib"
+    #sources:
+      #- type: archive
+        #url: http://mumps.enseeiht.fr/MUMPS_5.4.1.tar.gz
+        #sha256: 93034a1a9fe0876307136dcde7e98e9086e199de76f1c47da822e7d4de987fa8
+        #x-checker-data:
+          #type: anitya
+          #project-id: 5605
+          #stable-only: true
+          #url-template: http://mumps.enseeiht.fr/MUMPS_$version.tar.gz
+        ## Use Makefile from AUR
+      #- type: file
+        #url: https://aur.archlinux.org/cgit/aur.git/plain/Makefile.inc?h=mumps
+        #sha256: 978ad0fcb43665ac5fb2312ecc9fd1c22087e237a1793723eb51972132002c2b
+        #dest-filename: Makefile.inc
+
+  - name: Elmer
+    buildsystem: cmake
+    builddir: true
+    build-options:
+      env:
+        CFLAGS: '-L /app/lib -lMPI -lMPI::MPI_C'
+    config-opts:
+      - -DCMAKE_BUILD_TYPE:STRING=Release
```

```suggestion
      - -DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo
```

It's always nice to have the debug info